change how team assignment works for the Puppet module (#12566)
fleetdm/fleet: docs/Contributing/API-for-contributors.md

#### Parameters

| Name                     | Type    | In   | Description                                                                                  |
| ------------             | ------- | ---- | -----------------------------------------------------------                                  |
| external_host_identifier | string  | body | **Required**. The identifier of the host as generated by the external service (e.g. Puppet). |
| host_uuid                | string  | body | **Required**. The UUID of the host.                                                          |
| profile                  | string  | body | **Required**. The base64-encoded .mobileconfig content of the MDM profile.                   |
| group                    | string  | body | The group label associated with that profile. This information is used to generate team names if they need to be created. |

> | exclude                  | boolean | body | If the profile shouldn't be delivered to this host. |

Reviewer: For boolean properties, we've used the convention of starting descriptions with "Whether [...]". I realize that's a little tricky here since `true` means _don't_ do something...

Would it work to say something like:
```suggestion
| exclude                  | boolean | body | Whether to skip delivering the profile to this host. |
```

Author: much better, thanks!
